actions/contacts/search.go

```diff
@@ -72,3 +72,116 @@ func (a *Action) search(c *gin.Context) {
 
 	c.JSON(http.StatusOK, response)
 }
+
+// TODO: this method will be changed based on search poc
```

Comment:
Are you planning to do it in this PR or in some next task?
- If in the next task, is there a JIRA ticket for it?

Reply:
In next PR. This is [ticket number](https://bitcoinassociation.atlassian.net/browse/SPV-881)